products/db-cpdb/sql/_create_export.sql

```diff
@@ -1,11 +1,13 @@
 DROP TABLE IF EXISTS cpdb_dcpattributes_pts;
 SELECT * INTO cpdb_dcpattributes_pts
-FROM cpdb_dcpattributes 
-WHERE ccpversion = :'ccp_v' 
-AND ST_GeometryType(geom)='ST_MultiPoint';
+FROM cpdb_dcpattributes
+WHERE
+    ccpversion =: 'ccp_v'
+    AND ST_GEOMETRYTYPE(geom) = 'ST_MultiPoint';
 
 DROP TABLE IF EXISTS cpdb_dcpattributes_poly;
 SELECT * INTO cpdb_dcpattributes_poly
-FROM cpdb_dcpattributes 
-WHERE ccpversion = :'ccp_v' 
-AND ST_GeometryType(geom)='ST_MultiPolygon';
\ No newline at end of file
+FROM cpdb_dcpattributes
+WHERE
+    ccpversion =: 'ccp_v'
```

Comment:
This doesn't look like it's going to run

Reply:
Of note: https://github.com/sqlfluff/sqlfluff/pull/1709

Reply:
hm I wonder why it finds them unappeasable then

an example is `DROP TABLE IF EXISTS :TABLE` in `/db-pluto/pluto_build/sql/export_mappluto_gdb.sql`

Reply:
if it's breaking queries, I may have to set `fix_even_unparsable` back to the default of `False` and ignore all files with psql variables in them